src/main/scala/com/socrata/regioncoder/RegionCoderServlet.scala

```diff
@@ -1,32 +1,36 @@
 package com.socrata.regioncoder
 
+import javax.servlet.http.{HttpServletResponse => HttpStatus}
+
+import com.rojoma.json.v3.ast.{JObject, JString}
+import com.rojoma.json.v3.util.JsonUtil
 import com.socrata.geospace.lib.Utils._
 import com.socrata.regioncoder.config.RegionCoderConfig
 import com.socrata.soda.external.SodaFountainClient
-import javax.servlet.http.{HttpServletResponse => HttpStatus}
-import org.scalatra.{BadRequest, Ok, AsyncResult}
+import org.scalatra.metrics.MetricsSupport
+import org.scalatra.{AsyncResult, BadRequest, Ok}
 
+// scalastyle:off multiple.string.literals
 class RegionCoderServlet(rcConfig: RegionCoderConfig, val sodaFountain: SodaFountainClient)
-  extends RegionCoderStack with RegionCoder {
+  extends RegionCoderStack with RegionCoder with MetricsSupport {
 
   val cacheConfig = rcConfig.cache
   val partitionXsize = rcConfig.partitioning.sizeX
   val partitionYsize = rcConfig.partitioning.sizeY
 
+  def pointcodeTimer[A](f: => A): A = timer("pointcode") {f}.call()
+  def stringcodeTimer[A](f: => A): A = timer("stringcode") {f}.call()
+  def debugGetTimer[A](f: => A): A = timer("debug-get") {f}.call()
```

Comment:
I think we can skip putting timers on the debug routes. No one will never look at those metrics so it will just add noise.